src/site/_includes/components/Aside.js

```diff
@@ -119,6 +119,12 @@ function Aside(content, type = 'note') {
       icon = 'celebration.svg';
       title = i18n(`i18n.common.${type}`, locale);
       break;
+
+    case 'update':
+      icon = 'update.svg';
+      title = i18n(`i18n.common.${type}`, locale);
+      utilities.main = 'bg-primary-box-bg color-primary-box-text';
```

Comment:
I wonder if tying the color of this aside to "primary" color of the site is a good idea here? Let's say we update site's theme and now "primary" color was red, would we  still want this type of aside to be red? @tokyodanj wdyt?

Reply:
I created new variables for update aside theme named `bg-state-update-bg` and `color-state-update-text`.